src/containers/Projects/index.js

```diff
@@ -1,41 +1,66 @@
-
+/* @flow */
 
 import React, { Component, PropTypes } from 'react'
 import Helmet from 'react-helmet'
 import { push } from 'react-router-redux'
 import CircularProgress from 'material-ui/CircularProgress'
 import { ErrorMessage, ProjectList } from 'components'
+import GroupList from 'components/GroupList/GroupList'
 import { connect } from 'react-redux'
 import { fetchProjects } from 'ducks/projects'
 import { Sticky, StickyContainer } from 'react-sticky'
 import { Row, Col, Button, ButtonGroup } from 'react-bootstrap'
+import urljoin from 'url-join'
 
 export class Projects extends Component {
   componentDidMount() {
     const { dispatch } = this.props
     this.clickHandler = this.clickHandler.bind(this)
-    // todo: remove this from constructor
     dispatch(fetchProjects())
   }
 
   clickHandler(value) {
```

Comment:
All this url parsing doesn't seem right to me... As I haven't worked much with React Router, I'm not sure how this usually is handled. 

@kkateq ?